src/tracks/SampleTrack.cpp

```diff
@@ -310,11 +314,18 @@ void SampleTCO::loadSettings( const QDomElement & _this )
 		useCustomClipColor( true );
 		setColor( _this.attribute( "color" ) );
 	}
+
+	if(_this.hasAttribute("reversed"))
+	{
+		m_sampleBuffer->setReversed(true);
+		emit wasReversed();		// tell SampleTCOView to update the view
```

Comment:
I don't think we need double tabs here
```suggestion
		emit wasReversed(); // tell SampleTCOView to update the view
```